config/routes.rb

```diff
@@ -84,11 +84,19 @@
         resource :preview, only: [:update], controller: 'post_previews'
         member do
           get 'quote'
-          post 'mark_as_unread'
+          post 'mark_as_unread' # deprecated
```

Comment:
Perhaps we can just remove it, because most users would be using this method on PostView to generate the URL:

https://github.com/thredded/thredded/blob/235d0a9026c9f8248297c0fca13e655aca893a75/app/view_models/thredded/post_view.rb#L62